### PR TITLE
[WIP] Added Content to Checklist.md & Reformatted Protocols

### PR DESCRIPTION
**Pull request by growupboron**
### Summary
Added Content regarding the topic ```Checklist for managing and participating in a distributed team```

Expands the section mentioned in #960 and #861 

Solves #1110 

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks okay?

### CODE REVIEW

**Comment by growupboron** [comment]
@malvikasharan and @KirstieJane, Excited to hear your reviews?
Regards.

**Comment by malvikasharan** [comment]
Hi @growupboron, Thank you for your contributions. We definitely want to focus on creating a kind environment however that should not permeate through the entire checklist. This is intending to create step by step guide for managers. I will be able to look at this in details later this week, meanwhile, feel free to edit your contributions wherever you see fit.

**Comment by growupboron** [comment]
Sorry for the naiveness, Is there any particular author assigned to remote and distributed collaboration, and where can I find the working document? Regards.

**Comment by malvikasharan** [comment]
Not at all, I appreciate you taking the time to work on this. You can preview the existing sections here: https://deploy-preview-962--the-turing-way.netlify.app/remote_collaboration/
 You can see the other checklists to get an idea of how to structure the checklist you are working on.

**Comment by growupboron** [comment]
How can I view the netlify preview of this PR?

**Comment by malvikasharan** [comment]
You can view it here https://the-turing-way.netlify.com//remote_collaboration/remote_collaboration.html

**Comment by growupboron** [comment]
@malvikasharan from my understanding, that's the current deployed site. I am pretty sure that, only PR to master branch that prototyping of the PR and the link is given out by the netlify bot. In this case, it is not, or am i wrong? And netlify bot only checks/deploys the Ready to be reviewed PRs and not draft PRs?

**Comment by growupboron** [comment]
@malvikasharan Never mind. I set up the website build in my local setup. Did reformat the protocols page, should I open up a new PR, or let it remain with this?

**Reformatted** ```Protocols Page``` : 

Screenshot from 2020-05-17 10-52-24

**Comment by malvikasharan** [comment]
@growupboron If you can open an issue on protocols and connect that to this PR and reanme, it should be fine. I won't be able to review this week, but will ask other members to review.

**Comment by growupboron** [comment]
@malvikasharan Opened the Issue as requested.

File: book/content/remote_collaboration/checklist/checklist.md [hunk 1]
```diff
@@ -798,30 +798,58 @@ This checklist also aims to help participants support the event organisers by be
...
 <a name="tips-for-all-the-members"></a>
 ### Tips for all the members
-<details>
-  <summary>Click to expand</summary>
-
-</details>
+- [ ] Be courteous and have realistic exceptations from your colleagues  
+- [ ] Complete the tasks assigned to you
+- [ ] Always speak up your doubt to have a clear perspective on the team goals
+- [ ] If not able to attend a corresponding online meeting, send regards prior to the manager
```
Comment (c-martinez) [inline book/content/remote_collaboration/checklist/checklist.md:808]: Is 'send regards' really what is meant here? In my mind (maybe I'm wrong) 'send regards' is equivalent to 'say hi' -- and I guess you want to let the meeting organiser know you will not be able to make it.

Proposed rewording:
If not able to attend an online meeting, send your apologies to the meeting organiser

File: book/content/remote_collaboration/checklist/checklist.md [hunk 2]
```diff
@@ -798,30 +798,58 @@ This checklist also aims to help participants support the event organisers by be
...
+- [ ] Read meeting minutes prior and after the online meeting
+- [ ] Help fellow struggling colleagues
```
Comment (c-martinez) [inline book/content/remote_collaboration/checklist/checklist.md:810]: Proposed rewording:
Offer to help colleagues who you think might be struggling

File: book/content/remote_collaboration/checklist/checklist.md [hunk 3]
```diff
@@ -798,30 +798,58 @@ This checklist also aims to help participants support the event organisers by be
...
+- [ ] Be transparent with ideas, take and share appropriate credits
 
 <a name="specific-tips-for-the-managers"></a>
 ### Specific tips for the managers
-<details>
-  <summary>Click to expand</summary>
+<!---<details>
+  <summary>Click to expand</summary>--->
+
+- [ ] Clarify all Rules of Communication by relying on Right Remote Work Tools according to the demands of the team:
```
Comment (c-martinez) [inline book/content/remote_collaboration/checklist/checklist.md:818]: relying on Right --> relying on the Right

Comment (c-martinez) [reply book/content/remote_collaboration/checklist/checklist.md:818]: I believe there is a section under construction for "Choice of tools and security aspects" -- perhaps it is useful linking to that section (once it is available?)

File: book/content/remote_collaboration/checklist/checklist.md [hunk 4]
```diff
@@ -798,30 +798,58 @@ This checklist also aims to help participants support the event organisers by be
...
+  + [ ] Online Chatting Platforms like [Slack](https://slack.com/intl/en-in/),[Gitter](https://gitter.im),[Discord](https://discord.com/) or [IRC](https://en.wikipedia.org/wiki/Internet_Relay_Chat)
+  + [ ] Video Chatting Platforms like [Zoom](https://zoom.us/), [Google Meet](https://meet.google.com/), [Microsoft Teams](https://www.microsoft.com/en-in/microsoft-365/microsoft-teams/group-chat-software), [Skype](https://www.skype.com/en/) or the open source alternative [Jitsi](https://jitsi.org/)
+- [ ] Maintain a meeting minutes and video recording of every online-meeting. 
+- [ ] Respect time differences and assign tasks accordingly
+- [ ] Maintain a central information repository for the whole team
+- [ ] Instill a culture of constructive criticism, be open to new and bold ideas
+- [ ] Encourage and Embrace Diversity. Keep cultural differences in mind to better relate with participants
```
Comment (c-martinez) [inline book/content/remote_collaboration/checklist/checklist.md:825]: Embrace Diversity -- I don't think they need to be capitalised here, do they?

File: book/content/remote_collaboration/checklist/checklist.md [hunk 5]
```diff
@@ -798,30 +798,58 @@ This checklist also aims to help participants support the event organisers by be
...
+- [ ] Set Expectations Early and Clearly to set up clear project roles and responsibilities
```
Comment (c-martinez) [inline book/content/remote_collaboration/checklist/checklist.md:826]: Expectations Early and Clearly -- also not sure if they need to be capitalised.

File: book/content/remote_collaboration/checklist/checklist.md [hunk 6]
```diff
@@ -798,30 +798,58 @@ This checklist also aims to help participants support the event organisers by be
...
+- [ ] Discuss with team members their specific goals in the project to keep everyone accountable
+- [ ] Write down working schedule, holidays, meeting and other ground rules of teams
+- [ ] Have a regular One-on-One and work on team bonding, and track hours worked, attendance and other basic measures of productivity
```
Comment (c-martinez) [inline book/content/remote_collaboration/checklist/checklist.md:829]: Have a regular One-on-One --> Have regular one-on-one meetings

File: book/content/remote_collaboration/checklist/checklist.md [hunk 7]
```diff
@@ -798,30 +798,58 @@ This checklist also aims to help participants support the event organisers by be
...
 ### Specific tips for the team members
-<details>
-  <summary>Click to expand</summary>
+<!---<details>
+  <summary>Click to expand</summary>--->
+
+- [ ] Don’t conflate brief communications and clear communications
+- [ ] Don’t bombard your team with messages, use communication channels professionally
+- [ ] Establish a structured way of completing tasks, ask for help, if needed
+- [ ] Act diligintly and responsibly, giving benefit of doubt to colleagues
```
Comment (c-martinez) [inline book/content/remote_collaboration/checklist/checklist.md:844]: diligintly -> diligently

File: book/content/remote_collaboration/checklist/checklist.md (hunk 1 above)
Comment (growupboron) [reply book/content/remote_collaboration/checklist/checklist.md:808]: Yeah, my mistake. Correcting it.

File: book/content/remote_collaboration/checklist/checklist.md (hunk 3 above)
Comment (growupboron) [reply book/content/remote_collaboration/checklist/checklist.md:818]: > I believe there is a section under construction for "Choice of tools and security aspects" -- perhaps it is useful linking to that section (once it is available?)

Will link it once it is available.

**Comment by growupboron** [comment]
> I left a couple of comments -- nothing major, but it would be good to address them.

Made the corresponding changes, thanks for the review.

**Comment by malvikasharan** [comment]
@c-martinez You're a star! Thanks for the review, and @growupboron Thanks for integrating those. I will add a second review later this week. Cheers!

**Comment by c-martinez** [comment]
With pleasure @malvikasharan :-)

**Comment by malvikasharan** [comment]
hi @growupboron. I think that the checklist still needs some work. You don't need to do that if you don't have the time, but if you feel like giving it another shot, please check the Zapier guidebook to keep it informative: https://zapier.com/learn/remote-work/#download

**Comment by growupboron** [comment]
@malvikasharan Sorry for dropping off, have been quite busy the later part of the month with assignment deadline. I'll get back to this ASAP.

**Pull request closed without merging**